action/common.go

```diff
@@ -0,0 +1,76 @@
+package action
+
+import (
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+// These Flag* consts exist to make any changes to flags consistent across the project
+const (
+	// FlagDebug turn on debugging output
+	FlagDebug = "debug"
+	// EnvDebug turn on debugging output
+	EnvDebug = "DEBUG"
+	// FlagVaaSURL address of the VaaS host to query
+	FlagVaaSURL = "vaas-url"
+	// EnvVaaSURL address of the VaaS host to query
+	EnvVaaSURL = "VAAS_URL"
+	// FlagUser represents the user name for Auth
+	FlagUser = "user, u"
+	// EnvVaaSUser represents the user name for Auth
+	EnvVaaSUser = "VAAS_USER"
+	// FlagSecretKey client key for Auth
+	FlagSecretKey = "key, k"
+	// EnvVaaSKey client key for Auth
+	EnvVaaSKey = "VAAS_KEY"
+	// FlagDirector represents the director name
+	FlagDirector = "director"
+	// FlagAddr address of this backend
+	FlagAddress = "addr"
+	// FlagPort represents the port of this backend
+	FlagPort = "port, p"
+	// FlagDryRun only perform read operations, do not make any changes
+	FlagDryRun = "dry-run"
```

Comment:
I cant spot place where this is used.

Reply:
Well, this is embarrassing, it's not. I guess I'll remove it then.